src/components/CryptCard/index.tsx

```diff
@@ -0,0 +1,62 @@
+import {StyleSheet, Text, View, Image} from 'react-native';
+import React, {useMemo} from 'react';
+import crypto from '../../../metrics.json';
+import PercentChangeArrow from '../../../assets/icons/north_east_black_24dp.svg';
+import {
+  ContainerView,
+  PercentChangeNumber,
+  PercentChangeView,
+  ColView,
+  LeftColTextView,
+  RightColTextView,
+  CryptImage,
+} from './styles';
+
+const CryptCard = () => {
+  const percentageChange = (percentNumber: number) => {
+    const roundNumber =
+      Math.round((percentNumber + Number.EPSILON) * 100) / 100;
+    const negative = roundNumber < 0;
+
+    return (
+      <PercentChangeView>
+        <PercentChangeArrow
+          width={16}
+          height={16}
+          rotation={negative ? 180 : 0}
+          fill={negative ? 'red' : 'green'}
```

Comment:
use theme

Reply:
the colors are considered constants, the the is not only to switch beetween dark or light, it stores all the colors, in all the cases where you use colors you should use theme, storing the colors there, so if you want to change one in the future and is implemented in multiple parts, it will be easy

Reply:
the theme should be something like this:

src/utils/theme.ts

```
const theme = {
   colors: {
      white: "#ffffff",
      black: "#000000",
      ........
   }
}
```

Reply:
also check for the doc of styled components to see how to provide it to all the styled components